eshenayo/edgex-examples
# ci(workflows): Add helm as tar zipped file via release action

This action adds a release asset of the individual example, for consumption by automation of the helm chart only as a tar zipped file. This does not modify any core functionality.

**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x ] I have fully tested (add details below) this the new feature or bug fix (if not, why?)

## Testing Instructions


1. Create a new release
2. Observe that a tar zipped asset is added containing the examples helm directory
